client/my-sites/theme/theme-related-themes-card/index.jsx

```diff
@@ -0,0 +1,86 @@
+/** @ssr-ready **/
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import i18n from 'lib/mixins/i18n';
+import SectionHeader from 'components/section-header';
+import { getDetailsUrl } from 'my-sites/themes/helpers';
+
+var ThemeRelatedThemesCard = React.createClass( {
+
+	propTypes: {
+		currentTheme: React.PropTypes.string.isRequired
+	},
+
+	getRelatedThemes() {
+		var themes = [
+			{ name: 'Twenty Sixteen', slug: 'twentysixteen' },
+			{ name: 'Rowling', slug: 'rowling' },
+			{ name: 'Hemingway Rewritten', slug: 'hemingway-rewritten' },
+			{ name: 'Gazette', slug: 'gazette' },
+			{ name: 'Ecto', slug: 'ecto' },
+			{ name: 'Isola', slug: 'isola' },
+			{ name: 'Edin', slug: 'edin' },
+			{ name: 'Sela', slug: 'sela' },
+			{ name: 'Pique', slug: 'pique' },
+			{ name: 'Harmonic', slug: 'harmonic' },
+		];
+
+		var selectedThemes = [];
+
+		while( selectedThemes.length < 2 ) {
+			let randomThemeIndex = Math.floor(Math.random() * themes.length );
```

Comment:
IIRC @folletto originally suggested we determine related themes from somehow hashing the current theme, so there'd be deterministically always the same themes. This is also somewhat relevant for SSR (caching) and to avoid reconc errors. (Since it's only a temporary solution, we probably shouldn't waste too much time on it -- but maybe there's some readily available hash function, e.g. in lodash.)